Jasmine/Common/Models/GameInfo.swift

```diff
@@ -0,0 +1,6 @@
+struct GameInfo {
+    let levelName: String
+    let gameType: GameType
+    let gameMode: GameMode
+    let gameViewModel: BaseViewModelProtocol
```

Comment:
`BaseViewModelProtocol`, that means I will have to downcast if I segue to the specific game.